doc/deps.md

```diff
@@ -61,7 +62,7 @@ sudo apt-get install git cmake make g++ libboost-all-dev \
 #### Ubuntu 18.04 LTS
 
 ```bash
-sudo apt-get install git cmake make g++ libboost-all-dev \
+sudo apt-get install git cmake make g++ gcc-7-plugin-dev libboost-all-dev \
```

Comment:
I guess this code suggestion is a typo, but instead `libssl-dev` shall be replaced with `libssl1.0-dev` in the Ubuntu 18.04 install guide?

Reply:
On the other hand, wouldn't it be a better solution to install a newer version of nodejs by the users from the official PPA? See very simple instructions here:
https://github.com/nodesource/distributions

E.g. I have nodejs-11 installed from the official PPA and `libssl-dev` from the standard package repository. That way I did not have to degrade to `libssl1.0-dev`. Since other packages I use depend on `libssl-dev`, degrading was not an option for me.

Reply:
We can resolve this later. I merely wanted to make sure the human-readable guide and the script "guide" are in sync.

If you are a power user who understands conflicts of packages, you'll most likely find a way to fix it. I tended in the way of the guard that if someone doesn't *really* know how this works in the big picture could also get it going.

Also, using custom PPAs... Well... I think I will be a minority, unfortunately, there is an upcoming corporate regulation that we cannot add custom PPAs to our machine. :facepalm:

Reply:
Valid point, let's update the docs this way for now, I have applied your suggestions.